### ISSUE TITLE
Keepass support?

### ISSUE THREAD

**Issue by Gwend4l** [gnumdk/eolie#70]
### Feature

Although I really like the choice of relying on GNOME keyring/seahorse for password management, a good amount of people (including myself) are using [Keepass](http://keepass.info/) nowadays to store all their passwords. Among other nice things, Keepass is crossplatform, and you can open your password database on windows or android very easily.

On Firefox, the [Keefox](http://keefox.org/) extension can be used to connect Firefox to the background keepass instance, and to fill in login forms automatically. 

Could we imagine being able to use Keepass with Eolie someday, similarly to what Keefox is doing? Not sure what would be the technical implications of that, but I would be very interested :)

**Comment by gnumdk** [comment]
Eolie is using freedesktop secret service, not GNOME keyring.

Please ask for Keepass to support freedesktop secret storage, it was in the roadmap in the past: 
https://www.keepassx.org/news/2010/09/242

**Comment by Gwend4l** [comment]
> Eolie is using freedesktop secret service, not GNOME keyring.

I see, sorry for the mistake. Since I saw the passwords appear in seahorse, and since seahorse is still officially described as "a GNOME application for managing encryption keys and passwords in the GnomeKeyring", I had made the wrong assumption.

> Please ask for Keepass to support freedesktop secret storage, it was in the roadmap in the past:
https://www.keepassx.org/news/2010/09/242

Indeed, and the corresponding ticket is here: https://dev.keepassx.org/issues/266

And just to clarify, I was talking about [Keepass](http://keepass.info/), not [KeepassX](https://www.keepassx.org), which are different projects. But since the file format is the same, I could indeed switch to KeepassX to benefit from such feature.
